library/psa_crypto.c

```diff
@@ -3191,8 +3191,8 @@ static psa_status_t psa_rsa_verify( mbedtls_rsa_context *rsa,
     if( status != PSA_SUCCESS )
         return( status );
 
-    if( signature_length < mbedtls_rsa_get_len( rsa ) )
-        return( PSA_ERROR_BUFFER_TOO_SMALL );
```

Comment:
In commit "Fix signature size checks in psa_asymmetric_verify for RSA", `If the signature doesn't have the correct signature` is more easily understood when written as `If the signature doesn't have the correct length`, although it is less cute.

Reply:
I amended the commit message.